src/App.vue

```diff
@@ -11,7 +11,8 @@
         v-if="isSupportedBrowser"
         class="supportedbrowser--alert"
       >
-        Alert - unsupported browser
+        Your browser does not support the Superhero extension.
+        Tipping and commenting will be disabled.
```

Comment:
Can't we just use deeplinks to the webwallet if their browser is not supported?

Reply:
It is possible actually, is there an API for that listed?

Reply:
It seems however much bigger change for that. Still this is not really related to the PR, as it is for the "supported" ones.

Reply:
If we embed the wallet into the website the need of deeplinks would be only for mobile browsers

Reply:
https://github.com/aeternity/superhero-wallet/blob/932e1b6926699174bd1583f5247163ca09bef4f8/tests/pages/index.html